src/components/views/elements/Tooltip.tsx

```diff
@@ -17,7 +17,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
```

Comment:
the changes in this file look unrelated?

Reply:
Yeah, that's true, though discovered in the course of this work... while working on the test, my editor highlighted that the `this.tooltip` field is never used, so it seemed like a good fix to send along. Do you want it in a separate PR instead?

Reply:
Not at all, just wanted to raise in case it crept in as part of a separate changeset you wanted to commit